### DISCUSSION TITLE
how can i refresh of table by using the button that is outside of table

### DISCUSSION THREAD

**Discussion by virusek392** [KevinVandy/material-react-table#466] [General]
Hi, how can i refresh of table by using the button that is outside of table (<MaterialReactTable />)?

![obraz](https://user-images.githubusercontent.com/107401577/231291230-37887821-a159-473f-84af-2ef315ddde5a.png)

**Comment by KevinVandy** [comment]
Just render it outside the table. Sorry, I don't understand this question and why that would be hard.

**Reply by virusek392** [reply]
I mean how deselected it  by using outside function like it

```
const MyDeselect = () =>
{
..... How deselect here by using code ?>
}
.....
return()
<MaterialReactTable
columns={columns}
data={data}
......
/>
<button onclick="MyDeselect()">My Button </button >
```

Please write me example code for this solution.

**Reply by KevinVandy** [reply]
Take a look at using the `tableInstanceRef` prop https://www.material-react-table.com/docs/guides/table-state-management#access-the-underlying-table-instance-reference

You can call table apis from anywhere

**Reply by virusek392** [reply]
Ok, thx

**Comment by KevinVandy** [comment]
btw, I see much of your table is still in English, consider looking at the localization docs to have MRT be in your preferred language https://www.material-react-table.com/docs/guides/localization
